### PR TITLE
add latest announcement card

### PR DESCRIPTION
**Pull request by ArinCantCode**
https://uniwhales.atlassian.net/browse/FE-239

### CODE REVIEW

**Review by shan8851 — APPROVED** [review]

File: src/organisms/announcementCard/styles.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { Styled } from '../../theme';
+import { AnnouncementCardProps } from './types';
+
+export const AnnouncementCardContainer = Styled.div<Pick<AnnouncementCardProps, 'width'>>`
+    background: ${({ theme }) => theme.containerAndCardShades.BG_SHADE_PLUS_4};
+    box-shadow: ${({ theme }) => theme.dropShadow.REGULAR};
+    width: ${({ width }) => width};
```
Comment (goshatravin) [inline src/organisms/announcementCard/styles.tsx:7]: @ArinCantCode  card fully locked for a specific width, i think this page , just 100% support mobile view
Screen Shot 2022-06-19 at 10 16 12

File: src/organisms/announcementCard/AnnouncementCard.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  AnnouncementCardContainer, AnnouncementText, Content, Cover,
+} from './styles';
+import { AnnouncementCardProps } from './types';
+
+export const AnnouncementCard: FC<AnnouncementCardProps> = ({
+  coverUrl, releaseDate, title, purifiedContent, width,
+}) => (
+  <AnnouncementCardContainer width={width}>
+    <Cover coverUrl={coverUrl} />
+    <Content>
```
Comment (goshatravin) [inline src/organisms/announcementCard/AnnouncementCard.tsx:13]: @ArinCantCode no spacing into this component ? ask Michal to review this component please.

**Review by goshatravin — CHANGES_REQUESTED** [review]

File: src/organisms/announcementCard/styles.tsx (hunk 1 above)
Comment (ArinCantCode) [reply src/organisms/announcementCard/styles.tsx:7]: Can you clarify? Should 100% be a default value?

Comment (goshatravin) [reply src/organisms/announcementCard/styles.tsx:7]: @ArinCantCode yep, u gonna control the width into the page with a breakpoints

**Comment by ArinCantCode** [comment]
@goshatravin everything fixed

**Pull request merged**